lms/envs/common.py

```diff
@@ -3943,3 +3943,4 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 LRS_API_KEY = os.environ.get('LRS_API_KEY')
 LRS_QUEUE = 'edx.lms.core.default'
 DEFAULT_LMS_QUEUE = 'edx.lms.core.default'
+LRS_TIMEOUT = int(os.environ.get('LRS_TIMEOUT', 2))
```

Comment:
It's possible we might want a float here instead

Reply:
Sure. Updated now.